### ISSUE TITLE
from scryers to aldors (and vice versa?)

### ISSUE THREAD

**Issue by rmzV** [cmangos/wotlk-db#185]
today i checked "city of light" and it was working))
after completing this quest and followed up, i decided swith to aldors (to test their quests as well)

so i started to turn in items in quest:
http://www.wowhead.com/quest=10019/more-venom-sacs

now i am neutral with aldors.. but what next? probably there are was a prequest to start aldors quest chains?

**Comment by evil-at-wow** [comment]
As far as I remember, getting back to neutral with the Aldors should be enough to be able to do their quests. So you're saying you don't see any Aldor quest now?

**Comment by rmzV** [comment]
so no this quests from
http://www.wowhead.com/npc=18166/archmage-khadgar

    Neutral 15 [65] Allegiance to the Aldor
        Neutral 15 [65] Ishanah?

only
Marks of Kil'jaeden quest available from npc at aldor rise

**Comment by rmzV** [comment]
**Switching to Aldor**
"  Shattrath City introduction quests Neutral 15 [60] **A'dal**, Neutral 15 [60] **City of Ligh**t and Neutral 15 [60] **Allegiance to the Aldor** are not available.
    Other Shattrath City quests Neutral 15 [60] **Ishanah**, Neutral 15 [62] **A Cure for Zahlia** and Neutral 15 [62] **Restoring the Light** are available.
    Netherstorm quest line starting with Neutral 15 [67] **Assist Exarch Orelis** is available.
    Shadowmoon Valley quest line starting with Neutral 15 [70] **Karabor Training Grounds** is not available. "
source:
http://wowwiki.wikia.com/wiki/Aldor
so basically only 3 shatrath and nether storm quests must be available

**Comment by rmzV** [comment]
there are is 0 info about Aldor to Scryer conversion
http://wowwiki.wikia.com/wiki/Scryers
but i belive that logic must be the same - starath and netherstorm quests must be available

**Comment by rmzV** [comment]
trying to switch back to scryers ..  via [dampscale basilik eyes]
now i am neutral .. and quest dissapeared, 
but at neutral Magistrix Fyalenn doesnot give me signets quest.. i just see a blue quest mark

**Comment by rmzV** [comment]
probably problem is ^ be has +scryers rep at start
